handler/area_types.go

```diff
@@ -65,3 +65,48 @@ func (h *AreaTypes) Get(w http.ResponseWriter, r *http.Request) {
 
 	h.respond.JSON(ctx, w, http.StatusOK, resp)
 }
+
+// GetAreaTypeParents is the handler for GET /area-types/{area-type}/parents
+func (h *AreaTypes) GetAreaTypeParents(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	var request contract.GetAreaTypeParentsRequest
+	if err := schema.NewDecoder().Decode(&request, r.URL.Query()); err != nil {
+		h.respond.Error(
+			ctx,
+			w,
+			http.StatusBadRequest,
+			errors.Wrap(err, "failed to decode query parameters"),
+		)
+		return
+	}
+
+	cantabularResponse, err := h.ctblr.GetGeographyDimensions(ctx, request.Dataset)
```

Comment:
I don't understand why we are querying the `GetGeographyDimensions` endpoint.
Shouldn't we be querying the `dp-api-client-go` endpoint and create a GraphQL query?